pkgs/development/libraries/jellyfin-ffmpeg/default.nix

```diff
@@ -1,21 +1,32 @@
 { ffmpeg_6-full
-, nv-codec-headers-12
+, ffmpeg
+, pipewire
+, openal
 , chromaprint
 , fetchFromGitHub
 , lib
 }:
 
-(ffmpeg_6-full.override {
-  nv-codec-headers-11 = nv-codec-headers-12;
-}).overrideAttrs (old: rec {
+(ffmpeg_6-full
+.override { # As upstream will not update to 6.1, they will wait until ffmpeg 7 come out
+  openal = openal.override {
+    pipewire = pipewire.override {
+      ffmpeg = ffmpeg.overrideAttrs {
+        version = "6.0.1";
```

Comment:
This override happens "after" the source url in `src` is constructed using the original version,  that's probably why you're getting a hash failure. You either want to override the entire `src` as well, or use `import  ../ffmpeg/generic.nix { version = "6.0.1", hash = "..."}`.

Reply:
Like this?

Reply:
Nevermind, this is a red herring :man_facepalming: `ffmpeg` uses `finalAttrs`, so your previous code should be fine on this front:

```
nix-repl> ffmpeg_6.src.rev
"n6.0"

nix-repl> (ffmpeg_6.overrideAttrs { version = "6.1"; }).src.rev
"n6.1"
```

Reply:
Ok